jujuclient/accounts.go

```diff
@@ -62,37 +82,3 @@ func ParseAccounts(data []byte) (map[string]AccountDetails, error) {
 type accountsCollection struct {
 	ControllerAccounts map[string]AccountDetails `yaml:"controllers"`
 }
-
-// TODO(axw) 2016-07-14 #1603841
```

Comment:
Please add "fixes ..." to the PR, and mark fix committed when this lands. I've just assigned it to you.